src/frontend/magnify/src/views/profile/account/ProfileAccountView.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { ProfileAccountView } from './index';
+
+export default {
+  title: 'Views/Account/ProfileAccountView',
+  component: ProfileAccountView,
+} as ComponentMeta<typeof ProfileAccountView>;
+
+const Template: ComponentStory<typeof ProfileAccountView> = (args) => {
+  return (
+    <RouterProvider
```

Comment:
ditto

Reply:
Yes for the moment the package is too linked to the router. Gotta undo it